pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -521,6 +521,75 @@ class Build {
         return testStages
     }
 
+    def runUploadTestImage() {
+        List testList = buildConfig.TEST_LIST
+        def jobName = "CRIU_Image_Upload"
+        def uploadJobs = [:]
+        def runUploadImage = false
+        def jobCommonParams = getCommonTestJobParams()
+        int version = jobCommonParams['JDK_VERSIONS'] as Integer
+        def testPlatform = jobCommonParams['ARCH_OS_LIST']
+        if ( testPlatform == "x86-64_linux" && (version == 11 || version == 17)) {
+            testList.each { testType ->
+                if (testType  == 'dev.external') {
+                    runUploadImage = true
+                }
+            }
+        }
+        try {
+            // only trigger CRIU_Image_Upload if the pipeline contains xlinux dev.external test build and the job exists
+            if (runUploadImage && JobHelper.jobIsRunnable(jobName as String)) {
+                def aqaBranch = 'master'
```

Comment:
Are there plans to change `master ` to `main`? Can we not pass the branch and rely on the default for cases where we don't need to override it?

Reply:
> Are there plans to change master to main?

I am not aware of the plan to change it main any time soon. 

> Can we not pass the branch and rely on the default for cases where we don't need to override it?

This is the default case. The code is the same as https://github.com/ibmruntimes/ci-jenkins-pipelines/blob/9dde6e98c6152c798d61a5b331517de2c3721d65/pipelines/build/common/openjdk_build_pipeline.groovy#L342